src/locales/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "randomInput": "I don't understand",
+  "sendLocation": "Send Location",
```

Comment:
why is it stored separately? maybe move it to the `onboarding` object since it's used inside `OnboardingController`?